test/xrt_test/CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (C) 2024, Advanced Micro Devices, Inc. All rights reserved.
+
+set(XDNA_XRT_TEST xrt_test.elf)
+
+add_executable(${XDNA_XRT_TEST}
+  xrt_test.cpp
+  )
+
+target_compile_definitions(${XDNA_XRT_TEST} PRIVATE
+  # below macros is required so that i/f defined in ishim.h is
+  # consistent with native xrt implementation
+  XRT_ENABLE_AIE
+  XRT_XDNA_BUILD
+  )
+
```

Comment:
These compilation definitions can be removed. They impact shim level API definitions. XRT application does not need to specify them.